### PR TITLE
Adds or updates certain transition redirects

### PR DESCRIPTION
**Pull request by dorothyyeager**
Updates broader transition redirect for annual reports to go from /about-fec/reports/ to /resources/cms-content/documents/
Adds redirects:
Old EEO page /eeo/policy_statement.shtml to https://www.fec.gov/about/equal-employment-opportunity/
Old EEO directory (broader)	/eeo/(.*).pdf to https://www.fec.gov/about/equal-employment-opportunity/
Old Records on Transition (broader) /pdf/record/(.*).pdf to https://www.fec.gov/resources/cms-content/documents/resources/record/$1.pdf
Older version of sale or use brochure /pages/brochures/sale_and_use_brochure.pdf to https://www.fec.gov/updates/sale-or-use-contributor-information/
Old Requests for legal consideration	/law/legalconsideration.shtml to https://www.fec.gov/legal-resources/policy-other-guidance/requests-legal-consideration/

### CODE REVIEW

**Comment by kathycarothers** [comment]
@dorothyyeager  In my notes on the sheet it looks we need to include this one too for transition:  rewrite ^/pdf/firsttenyears.pdf https://www.fec.gov/resources/cms-content/documents/firsttenyears.pdf redirect;

**Review by kathycarothers — APPROVED** [review]

File: redirects-transition.conf [hunk 1]
```diff
@@ -83,7 +83,8 @@ rewrite ^/disclosureie/ienational.do?candOffice=S https://www.fec.gov/data/indep
 rewrite ^/disclosurep/pnational.do https://www.fec.gov/data/candidates/president/presidential-map/ redirect;
 
 # eeo/ redirects
-rewrite ^/eeo/eeo.shtml https://www.fec.gov/about/equal-employment-opportunity/ redirect; 
+rewrite ^/eeo/eeo.shtml https://www.fec.gov/about/equal-employment-opportunity/ redirect;
+rewrite ^/eeo/policy_statement.shtml https://www.fec.gov/about/equal-employment-opportunity/ redirect;
```
Comment (patphongs) [inline redirects-transition.conf:87]: @dorothyyeager Should we be more specific with the EEO policy statement redirect to go to the anchor here: https://www.fec.gov/about/equal-employment-opportunity/#policies-and-statements

**Review by patphongs — COMMENTED** [review]
@dorothyyeager One small comment:

File: redirects-transition.conf (hunk 1 above)
Comment (dorothyyeager) [reply redirects-transition.conf:87]: I think so. I'll update my PR.

**Comment by lbeaufort** [comment]
Note that the builds will fail for this branch because it's older than the CircleCI changes. It should be fine to merge, or we could rebase off `develop` (that's something Pat or I could help with)

**Comment by patphongs** [comment]
There were some rebase issues with this PR, closing in favor of this one: https://github.com/fecgov/fec-proxy/pull/201

**Pull request closed without merging**